rc/tools/comment.kak

```diff
@@ -10,8 +10,9 @@ declare-option -docstring "characters inserted after a commented block" \
 
 # Default comments for all languages
 hook global BufSetOption filetype=asciidoc %{
-    set-option buffer comment_block_begin '///'
-    set-option buffer comment_block_end '///'
```

Comment:
C.f. my comment above, this might be correct?

Reply:
Nevermind, it seems to be 4.

https://github.com/asciidoc/asciidoc/blob/0beddc73873acb3724b3cb93999acf0ddba84fc5/asciidoc.conf#L304-L308